cmd/headscale/cli/mockoidc.go

```diff
@@ -63,6 +65,29 @@ func mockOIDC() error {
 		}
 		accessTTL = newTTL
 	}
+	
+	mockUsers := os.Getenv("MOCKOIDC_USERS")
```

Comment:
Would be great if this is prefixed with "HEADSCALE_INTEGRATION_"

Reply:
This is (technically) not just useful for integration tests, as this environment variable controls what mockoidc does if you e.g. set the env variable and ran `headscale mockoidc` yourself. I followed the naming of the other variables in the `mockOIDC` function (`MOCKOIDC_CLIENT_ID`, `MOCKOIDC_CLIENT_SECRET`, and so on). I can still prefix this if desired.